utils/profile-actions.js

```diff
@@ -15,13 +15,18 @@ export function fillProfile(
 
   cy.get('#client-last-name').type(options.lastName, { force: true })
 
-  cy.get('#client-document').type('00759459169', { force: true })
+  cy.get('#client-document').type(getDocument(), { force: true })
 
   cy.get('#client-phone').type('21999999999', { force: true })
 
   cy.get('#go-to-shipping').click()
 }
 
+export function getDocument() {
```

Comment:
Why don't we generate a fully random CPF instead? I think the code below does the job:
```js
// Generates a random array of 9 digits
const cpf = [...new Array(9)].map(() => Math.floor(10 * Math.random()))
// Adds the 10-th digit
cpf.push(cpf.reduce((sum, value, idx) => sum + (10 - idx) * value, 0) * 10 % 11)
// Adds the 11-th digit
cpf.push(cpf.reduce((sum, value, idx) => sum + (11 - idx) * value, 0) * 10 % 11)
// Transforms array into string
return cpf.join('')
```

Reply:
Because I wanted to validate if the document is really the responsible for Masterdata errors. The "other approach" I said on description is to use a document generator.

Reply:
Since you already had the job to add it, I'm ok to use it. Great job!